src/common/components/DataView.tsx

```diff
@@ -0,0 +1,60 @@
+import { DataObject } from '../types/NarrativeDoc';
+import { FC } from 'react';
+import { getWSTypeName } from '../utils/stringUtils';
+import classes from './DataView.module.scss';
+import TypeIcon from './TypeIcon';
+
+interface DataViewProps {
+  accessGroup: number;
```

Comment:
I'm not clear on what accessGroup is at first reading. Is there a clearer name for this prop? Or maybe a comment explaining what it is?